src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -73,6 +79,7 @@ export default class SpotTileModel extends ModelBase {
     this.executionConnected = false;
     this.isTradeExecutionInFlight = false;
     this._regionManagerHelper = new RegionManagerHelper(RegionNames.workspace, regionManager, this);
+    this.openFinAvailable = openFin.available;
```

Comment:
looks like `open.available` is a bug, will always be true. 

Can you remove this property on the openFin object (any anywhere using it) and use `openFin.isRunningInOpenFin`?